modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -139,6 +139,41 @@ definition {
 			actionName = "Action Name");
 	}
 
+	@description = "LPS-145665 - Verify that you can edit the Action name"
+	@priority = "4"
+	test CanEditActionName {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 145",
+			objectName = "CustomObject145",
+			pluralLabelName = "Custom Objects 145");
```

Comment:
We should standardize our naming convention for the custom objects in all of our tests, for now we can use the ticket number, like we do in other files.